### PR TITLE
[prototype] Remove `_FT` aliases from functional

### LINKED ISSUE #6818: Performance improvements for transforms v2 vs. v1
In addition to a lot of other goodies that transforms v2 will bring, we are also actively working on improving the performance. This is a tracker / overview issue of our progress.

Performance was measured with this [benchmark script](https://gist.github.com/pmeier/e0f1ea77c9cf75b682d7f30366a89bf8). Unless noted otherwise, the performance improvements reported above were computed on uint8, RGB images and videos while running single-threaded on CPU. You can find the full benchmark results alongside the benchmark script. The results will be constantly updated if new PRs are merged that have an effect on the kernels.

Kernels: 

- color
  - [x] `adjust_brightness` #6784
  - [x] `adjust_contrast` #6784 #6933
  - [x] `adjust_gamma` #6820 #6903
  - [x] `adjust_hue` #6805 #6903 #6938
  - [x] `adjust_saturation` #6784 #6940
  - [x] `adjust_sharpness` #6784 #6930
  - [x] `autocontrast` #6811 #6935 #6942
  - [x] `equalize` #6738, #6757, #6776
  - [x] `invert` #6819
  - [x] `posterize` #6823, #6847
  - [x] `solarize` #6819
- geometry
  - [x] `affine` #6949
  - [x] `center_crop` #6880 #6949
  - [x] `crop` #6949
  - [x] `elastic` #6942
  - [x] `erase` #6983
  - [x] `five_crop`: Composite kernel #6949
  - [x] `pad` #6949
  - [x] `perspective` #6907 #6949
  - [x] `resize` #6892
  - [x] `resized_crop`: Composite kernel #6892 #6949
  - [x] `rotate` #6949
  - [x] `ten_crop`: Composite kernel #6949
- meta
  - [x] `convert_color_space` #6784 #6832
  - [x] `convert_dtype` #6795 #6903
    - There is still some performance gain left for `int` to `int` conversion. Currently, we are using a multiplication 
      but theoretically bit shifts are faster. However, on PyTorch core the CPU kernels for bit shifts are not 
      vectorized making them slower for regular sized images than a multiplication. pytorch/pytorch#88607
- misc
  - [x] `gaussian_blur` #6762 #6888
  - [x] `normalize` #6821

Transform Classes:

- [x] MixUp/CutMix #6835
- [x] ColorJitter, RandomPhotometricDistort #6837

C++ (PyTorch core):

- [x] `vertical_flip` #6983 https://github.com/pytorch/pytorch/pull/89414
- [x] `horizontal_flip` #6983 https://github.com/pytorch/pytorch/pull/88989 https://github.com/pytorch/pytorch/pull/89414

cc @vfdev-5 @datumbox @bjuncek

### PR DESCRIPTION
**Pull request by datumbox**
Closes #6818

This PR eliminates the use of `_FT` aliases from prototype functional by porting over the last 3 remaining kernels. There might be some trivial speed improvements due to the elimination of unnecessary checks but nothing worth measuring IMO.

cc @vfdev-5 @bjuncek @pmeier

### CODE REVIEW

File: torchvision/prototype/transforms/functional/_color.py [hunk 1]
```diff
@@ -18,10 +19,12 @@ def adjust_brightness_image_tensor(image: torch.Tensor, brightness_factor: float
     if brightness_factor < 0:
         raise ValueError(f"brightness_factor ({brightness_factor}) is not non-negative.")
 
-    _FT._assert_channels(image, [1, 3])
+    c = image.shape[-3]
+    if c not in [1, 3]:
+        raise TypeError(f"Input image tensor permitted channel values are {[1, 3]}, but found {c}")
```
Comment (pmeier) [inline torchvision/prototype/transforms/functional/_color.py:24]:
```suggestion
        raise TypeError(f"Input image tensor permitted channel values are 1 or 3, but found {c}")
```

**Review by pmeier — APPROVED** [review]
One minor nit otherwise LGTM!

File: torchvision/prototype/transforms/functional/_color.py (hunk 1 above)
Comment (datumbox) [reply torchvision/prototype/transforms/functional/_color.py:24]: This is the error we have in all other methods, check the rest of the methods.

Comment (pmeier) [reply torchvision/prototype/transforms/functional/_color.py:24]: Didn't see that sorry. Yes, we should change it everywhere. Dynamic string creation does not make sense if the value is static. You can do it here, but I'm also ok to do that in a follow-up.

**Review by vfdev-5 — APPROVED** [review]
LGTM

File: torchvision/prototype/transforms/functional/_color.py (hunk 1 above)
Comment (datumbox) [reply torchvision/prototype/transforms/functional/_color.py]: All good. I did it here for all.

**Pull request merged**